### ISSUE TITLE
Error in logs

### ISSUE THREAD

**Issue by Kedryn** [hassio-addons/addon-tautulli#59]
I get this in the HA log

> -----------------------------------------------------------
>  Add-on: Tautulli
>  Monitoring and tracking tool for Plex Media Server
> -----------------------------------------------------------
>  Add-on version: 1.5.0
>  **There is an update available for this add-on!
>  Latest add-on version: null
>  Please consider upgrading as soon as possible.**
>  System: Ubuntu 18.04.4 LTS  (amd64 / qemux86-64)
>  Home Assistant Core: 0.110.2
>  Home Assistant Supervisor: 224

And also:

> 2020-05-26 12:59:16 - INFO :: MainThread : Checking if the database upgrades are required...
> 2020-05-26 12:59:16 - INFO :: MainThread : Checking if configuration upgrades are required...
> 2020-05-26 12:59:16 - DEBUG :: MainThread : Trying to execute: "git rev-parse HEAD" with shell in /opt
> 2020-05-26 12:59:16 - DEBUG :: MainThread : Git output: c825176563a21245a035514b1d870aa814d77f63
> 2020-05-26 12:59:16 - DEBUG :: MainThread : Trying to execute: "git rev-parse --abbrev-ref --symbolic-full-name @{u}" with shell in /opt
> 2020-05-26 12:59:16 - DEBUG :: MainThread : Git output: fatal: HEAD does not point to a branch
> 2020-05-26 12:59:16 - ERROR :: MainThread : Git returned bad info. Are you sure this is a git installation?
> 2020-05-26 12:59:16 - ERROR :: MainThread : Could not retrieve remote name from git. Falling back to origin.
> 2020-05-26 12:59:16 - ERROR :: MainThread : Could not retrieve branch name from git. Falling back to master.

**Comment by sinclairpaul** [comment]
What issue is being logged here?, just that the messages appear?

**Comment by Kedryn** [comment]
Yep, i wasn't able to get to the Tautulli interface, so i gave a look in the logs.

Anyway, later, i've found out the problem. Tautulli was responding on the local ip (i.e. 192.168.0.48) but not to the domain name (xxx.duckdns.org) that was the same ip.

I managed to get into the tautulli config web page, modify the public address cahnging it from ip to domain url, and then it worked also with the domain name.

If you are not interested in those error, you can close this issue.
